### ISSUE TITLE
Table checkbox data-on-check-click

### ISSUE THREAD

**Issue by rooks-1** [olton/Metro-UI-CSS#1476]
When clicking a button to trigger table checkbox, data-on-check-click doesn't fire.

When manually clicking the checkbox in table, the event fires.

Is there a way to trigger the checkbox event in code?

 ```
   <table data-role="table" class="table" data-check="true" data-show-pagination="false"
        data-show-search="false" data-show-table-info="false" data-show-rows-steps="false"
        data-on-check-click="checkClick">
        <thead>
            <tr>
                <td>ColA</td>
                <td>ColB</td>
            </tr>
        </thead>
        <tbody>
            <tr>
                <td>A</td>
                <td>B</td>

            </tr>
        </tbody>
    </table>

    <button onclick="buttonclick()">checkbox</button>
```

```
    <script>

        function checkClick() {
            console.log("checkbox clicked....")
        }

        function buttonclick() {
            $("input[type=checkbox]").click();
        }

    </script>
```

Tested using Metro 4.3.4 in Chrome

**Comment by olton** [comment]
fixed in 4.3.5, thanks

**Comment by rooks-1** [comment]
Thanks, tested and works in 4.3.5
